contents/javascript/Javascript-promises.md

```diff
@@ -0,0 +1,159 @@
+# Javascript: Promises
```

Comment:
Better than before 👍 Some way to go before you can make it understandable to a 'knowledgeable outsider' (not only to existing JS programmers). I suspect you can some inspiration from this article https://scotch.io/tutorials/javascript-promises-for-dummies (remember to acknowledge it).

Reply:
Added a snippet to acknowledge their example.